core/state/statedb.go

```diff
@@ -1362,7 +1370,7 @@ func (s *StateDB) deleteStorage(addr common.Address, addrHash common.Hash, root
 	)
 	for it.Next(true) {
 		// arbitrary stateSize limit, make it configurable
-		if stateSize+nodeSize > 512*1024*1024 {
+		if stateSize+nodeSize > storageDeleteLimit {
```

Comment:
It is better to skip handling destructions than to change the threshold to 0.

Reply:
<img width="795" alt="image" src="https://github.com/bnb-chain/bsc/assets/122502194/f3ffd550-b490-4598-a1be-43261ab46e94">
ok. go-ethereum has skip it now